tests/expectations/metrics/test_core.py

```diff
@@ -7,9 +7,13 @@
 import numpy as np
 import pandas as pd
 import pytest
+from great_expectations.data_context.types.base import (
+    DataContextConfig,
+    FilesystemStoreBackendDefaults,
+)
 
 import great_expectations.exceptions as gx_exceptions
-from great_expectations.core.batch import Batch
+from great_expectations.core.batch import Batch, RuntimeBatchRequest
```

Comment:
@TrangPham @Shinnnyshinshin Here are some thoughts: First, I feel that we should be reproducing this error using Fluent Datasources (FDS) and debugging it with FDS as well.  There is no benefit in supporting `RuntimeBatchRequest` if we do not also solve the issue for the FDS based approach as well.  Second, I believe that tests with `BatchRequest` should not be in the `test_core.py` module on the account of the fact that this module is for the lower level -- metrics -- scenarios.  I am happy to brainstorm both of these points.  Thank you.

Reply:
Where should we move the test? 

Do you have an example of similar test in FDS?